test/e2e/src/core/e2e_tests.sh

```diff
@@ -51,18 +51,67 @@ waitForArcK8sClusterCreated() {
     echo "Arc K8s cluster connectivityState: $connectivityState"
 }
 
+waitForLCMCreated() {
+    connectivityState=false
+    max_retries=60
+    sleep_seconds=10
+    for i in $(seq 1 $max_retries)
+    do
+      echo "iteration: ${i}, clustername: ${CLUSTER_NAME}, resourcegroup: ${RESOURCE_GROUP}"
+      ## hybridaks does not support checking connectivityStatus  maybe compare date with system date to see if has difference
+      # az hybridaks show -n $k8sClusterName -g $resourceGroup --query properties.status.featuresStatus.arcAgentStatus.lastConnectivityTime -o json
+
+      clusterState=$(az connectedk8s show --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP --query connectivityStatus -o json)  arc
```

Comment:
i see "arc"  in line 64.  looks like its typo.

Reply:
this one currently the hybridaks command does not support the connectivityStatus parameter, and this query needs to  be replaced by the hybridaks command to check the cluster status

Reply:
I am referring "arc"  at the end of the line.